app/views/admin/broadcasts/index.html.erb

```diff
@@ -1,20 +1,20 @@
-<nav class="flex mb-4" aria-label="Broadcasts navigation">
-  <ul class="nav nav-pills">
-    <li class="nav-item">
-      <%= link_to "All", admin_broadcasts_path, class: "nav-link #{'active' if params[:type_of].blank?}" %>
+<header class="flex items-center mb-6">
+  <h2 class="crayons-title">Broadcasts</h2>
+  <a href="<%= new_admin_broadcast_path %>" class="crayons-btn ml-auto" role="button">Make a New Broadcast</a>
+</header>
+
+<nav class="mb-2 flex justify-between items-center" aria-label="Broadcasts navigation">
+  <ul class="crayons-navigation crayons-navigation--horizontal">
+    <li>
+      <a href="<%= admin_broadcasts_path %>" class="crayons-navigation__item <%= "crayons-navigation__item--current" if params[:type_of].blank? %>">All</a>
     </li>
-    <li class="nav-item">
-      <%= link_to "Announcement", admin_broadcasts_path(type_of: :announcement), class: "nav-link #{'active' if params[:type_of] == 'announcement'}" %>
+    <li>
+      <a href="<%= admin_broadcasts_path(type_of: :announcement) %>" class="crayons-navigation__item <%= "crayons-navigation__item--current" if params[:type_of] == "announcement" %>">Announcements</a>
```

Comment:
Question: @rt4914 is there a reason we change from `link_to` to an `a` tag in this file?

Reply:
The only reason was keeping it consistent with what Pawel did in https://github.com/forem/forem/pull/16359 PR in `app/views/admin/articles/index.html.erb`.

Should I revert it back to using `link_to` ? Also, what is the major difference between the two?

Reply:
In general, it's usually more efficient when having complex paths and saves us from messy interpolations with html and erb. However, in this scenario there's no huge difference because the path is a straightforward one, however for consistency purposes I'm of the opinion that it makes more sense to use the `link_to` helper.

Reply:
Thanks for the clarification. I have used `link_to` instead of `a` tag now.